tests/examples/test_contrib/test_sqlalchemy/test_sqlalchemy_examples.py

```diff
@@ -1,14 +1,32 @@
+from pathlib import Path
+
 import pytest
+from pytest import MonkeyPatch
+from sqlalchemy.ext.asyncio import create_async_engine
+from sqlalchemy.pool import NullPool
 
+from litestar.plugins.sqlalchemy import AsyncSessionConfig, SQLAlchemyAsyncConfig
 from litestar.testing import TestClient
 
 pytestmark = pytest.mark.xdist_group("sqlalchemy_examples")
 
 
-def test_sqlalchemy_declarative_models() -> None:
-    from docs.examples.contrib.sqlalchemy.sqlalchemy_declarative_models import app
+async def test_sqlalchemy_declarative_models(tmp_path: Path, monkeypatch: MonkeyPatch) -> None:
```

Comment:
Without this additional monkey-patching, there were intermittent failures on this job.  This is the only way I could make the tests consistently pass on all python versions.